cmd/create/execution_test.go

```diff
@@ -147,20 +180,65 @@ func TestCreateLaunchPlanExecutionFunc(t *testing.T) {
 	tearDownAndVerify(t, `execution identifier project:"flytesnacks" domain:"development" name:"f652ea3596e7f4d80a0e"`)
 }
 
-func TestCreateTaskExecutionFunc(t *testing.T) {
+func TestCreateRelaunchExecutionFunc(t *testing.T) {
 	setup()
 	createExecutionSetup()
-	executionCreateResponseTask := &admin.ExecutionCreateResponse{
+	executionCreateResponseLP := &admin.ExecutionCreateResponse{
 		Id: &core.WorkflowExecutionIdentifier{
 			Project: "flytesnacks",
 			Domain:  "development",
-			Name:    "ff513c0e44b5b4a35aa5",
+			Name:    "f652ea3596e7f4d80a0e",
 		},
 	}
-	mockClient.OnCreateExecutionMatch(ctx, mock.Anything).Return(executionCreateResponseTask, nil)
-	executionConfig.ExecFile = testDataFolder + "task_execution_spec.yaml"
+	literalMap := &core.LiteralMap{
+		Literals: nil,
+	}
+	exec = &admin.Execution{
+		Id: &core.WorkflowExecutionIdentifier{
+			Project: config.GetConfig().Project,
+			Domain:  config.GetConfig().Domain,
+			Name:    "ffb31066a0f8b4d52b77",
+		},
+		Spec: &admin.ExecutionSpec{
+			LaunchPlan: &core.Identifier{
+				Name:    "core.advanced.run_merge_sort.merge_sort",
+				Version: "v3",
+			},
+			Inputs: literalMap,
+		},
+	}
+	launchPlan = &admin.LaunchPlan{
+		Id: &core.Identifier{
+			Name:    "core.advanced.run_merge_sort.merge_sort",
+			Version: "v3",
+		},
+	}
+	mockFetcher = &mocks.Fetcher{}
+	cmdGet.DefaultFetcher = mockFetcher
+	mockFetcher.OnFetchExecutionMatch(mock.Anything, mock.Anything, mock.Anything, mock.Anything, mock.Anything).Return(exec, nil)
+	mockFetcher.OnFetchLPVersionMatch(mock.Anything, mock.Anything, mock.Anything, mock.Anything, mock.Anything, mock.Anything).Return(launchPlan, nil)
+	mockClient.OnCreateExecutionMatch(ctx, mock.Anything).Return(executionCreateResponseLP, nil)
+	executionConfig.Relaunch = "xb5317xbty"
```

Comment:
is this the execution to be relaunched? how come it doesn't match "ffb31066a0f8b4d52b77",?

Reply:
Since i am using mock.Anything as input hence it didnt require a match . Have added an exact match now